src/ghga_service_commons/httpyexpect/server/handlers/fastapi_.py

```diff
@@ -14,9 +14,7 @@
 # limitations under the License.
 #
 
-"""
-Handeling exceptions in FastAPI. FastAPI has to be installed.
-"""
+"""Handeling exceptions in FastAPI. FastAPI has to be installed."""
```

Comment:
```suggestion
"""Handling exceptions in FastAPI. FastAPI has to be installed."""
```

Reply:
Fixed in [43d7907](https://github.com/ghga-de/ghga-service-commons/pull/20/commits/43d7907b0919b6d7b1ad544cb1ce37156681f6ae)